Subject: Fix dependency version on SHA
From: Giovanni Mascellani <gio@debian.org>
Forwarded: yes

The cabal description file declares a dependency against SHA < 1.5,
which is not needed because Gitit is able to use version 1.5.0.0
(present in Debian at the moment) too. This patch relaxes the
versioned dependency.

Index: gitit-0.8/gitit.cabal
===================================================================
--- gitit-0.8.orig/gitit.cabal	2011-05-27 18:34:42.000000000 +0200
+++ gitit-0.8/gitit.cabal	2011-05-27 18:35:11.000000000 +0200
@@ -130,7 +130,7 @@
                      bytestring,
                      random,
                      utf8-string >= 0.3 && < 0.4,
-                     SHA > 1 && < 1.5, HTTP >= 4000.0 && < 4000.2,
+                     SHA > 1, HTTP >= 4000.0 && < 4000.2,
                      HStringTemplate >= 0.6 && < 0.7,
                      network >= 2.1.0.0 && < 2.4,
                      old-locale >= 1,
